bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/SvgImageCreatorImp.java

```diff
@@ -30,22 +31,22 @@ public class SvgImageCreatorImp implements SvgImageCreator {
     private SwfSlidesGenerationProgressNotifier notifier;
     private long imageTagThreshold;
     private long pathsThreshold;
-    private int convPdfToSvgTimeout = 60;
+    private int convPdfToSvgTimeout = 3;

```

Comment:
```suggestion
    private int convPdfToSvgTimeout = 60;
```

Let's keep it 60 and change the timeout only for `detectFontType3`